Makefile

```diff
@@ -167,6 +167,14 @@ start_load_balancer:
 stop_load_balancer:
 	id=`podman ps --quiet --filter "name=load_balancer"`; test ! -z "$$id"  && podman rm -f load_balancer ; rm -f  $(HOME)/.test-infra/etc/nginx/stream.d/*.conf >& /dev/null || /bin/true
 
+########
+# Nat  #
+########
+
+stop_nat:
```

Comment:
can we have some explanation ? it looks like magic

Reply:
It removes all iptables nat rules having the mark 555.  Look at nat_controller.py